Modules/ProfileManager/ProfileManager.lua

```diff
@@ -235,6 +253,99 @@ function ProfileManager:AddTarget()
     end
 end
 
+function ProfileManager:PruneBelowLevel(minLevel, nop)
+    LOG:Trace("ProfileManager:PruneBelowLevel()")
+    local log = PruneLog:New("level", nop)
+    local prune
+    if nop then
+        LOG:Info("Prunning: No operation")
+        prune = (function(GUID, _log)
+            local profile = self.cache.profiles[GUID]
+            if not profile then return end
+            _log:Add(profile:Name())
+        end)
```

Comment:
Note that at point of declaration the log will be available inside the function.
```suggestion
        LOG:Info("Pruning: No operation")
        prune = (function(GUID)
            local profile = self.cache.profiles[GUID]
            if not profile then return end
            log:Add(profile:Name())
        end)
```